clang/test/Driver/sycl-offload-with-split.c

```diff
@@ -296,6 +296,16 @@
 // RUN:    | FileCheck %s -check-prefixes=CHK-NO-SPLIT
 // CHK-NO-SPLIT-NOT: sycl-post-link{{.*}} -split{{.*}}
 
+// Check no device code split mode is passed to sycl-post-link when -fsycl-device-code-split is not set but -fintelfpga is set
+// RUN:   %clang -### -fsycl -fintelfpga %s 2>&1 | FileCheck %s -check-prefixes=CHK-NO-SPLIT
```

Comment:
A couple suggestions to generalize the test/test description:
```suggestion
// Check no device code split mode is passed to sycl-post-link when -fsycl-device-code-split is not set but the FPGA triple is set
// RUN:   %clang -### -fsycl -fsycl-targets=spir64_fpga-unknown-unknown-sycldevice %s 2>&1 | FileCheck %s -check-prefixes=CHK-NO-SPLIT
```

Reply:
Fixed. Thanks for the suggestion.